views/partials/currently-playing.ejs

```diff
@@ -43,4 +45,26 @@
       {{user}}
     </span>
   </div>
+
+  <!-- CHAT -->
+  <div class="extra content">
+    <span style="cursor: pointer;" ng-click="toggleChat()">
+      {{chats.length}} message{{chats.length != 1 ? 's' : ''}}
+      <i class="blue chat caret right icon"></i>
+      <i class="blue chat caret down icon" style="display: none;"></i>
+    </span>
+  </div>
+  <div class="chat extra content" style="display: none;">
+    <div id="chat-list" class="ui relaxed list" style="overflow-y: auto;">
```

Comment:
What about combining the "listeners" list into some extra content at the top of the chat box? Right now it takes two clicks to see both messages and people listening, aka chatters.